e2e-tests/playwright/e2e/plugins/ocm.spec.ts

```diff
@@ -12,7 +12,7 @@ const clusterDetails = {
   status: 'Ready',
   platform: 'IBM',
   cpuCores: /CPU cores\d+/,
-  memorySize: /Memory size\d+ Gi/,
+  memorySize: /Memory size\d+ (Gi|Mi)/,
```

Comment:
```suggestion
  memorySize: /Memory size\d.*(Gi|Mi)/,
```

I saw when Mi is there was no space between the number and Mi, it could be a small UI issue 🤷‍♂️

Reply:
I'm not sure, but your suggestion is welcome 👍